ls1intum/Artemis
# show username in start view

### Checklist

### Motivation and Context


So simplify the process of entering the name, the student's name is shown in a greeting at the beginning


### Steps for Testing


1. create an exam with the visibility date before and the startDate after the current time
2. Manually create exams for all the participants
3. click on the exam to get to the start view
4. You should be greeted in english/german with your name and the exam title

### Screenshots



![image](https://user-images.githubusercontent.com/44401560/85951808-e69b6300-b965-11ea-9cdb-8e7b6ce73651.png)

EDIT: removed the "zur Klausur" or in english " to the exam" to make it shorter and more terse
